fdbserver/GrvProxyServer.actor.cpp

```diff
@@ -112,7 +114,13 @@ struct GrvProxyStats {
 	                          SERVER_KNOBS->LATENCY_METRICS_LOGGING_INTERVAL,
 	                          SERVER_KNOBS->LATENCY_SAMPLE_SIZE),
 	    recentRequests(0), lastBucketBegin(now()),
-	    bucketInterval(FLOW_KNOBS->BASIC_LOAD_BALANCE_UPDATE_RATE / FLOW_KNOBS->BASIC_LOAD_BALANCE_BUCKETS) {
+	    bucketInterval(FLOW_KNOBS->BASIC_LOAD_BALANCE_UPDATE_RATE / FLOW_KNOBS->BASIC_LOAD_BALANCE_BUCKETS),
+	    grvConfirmEpochLiveDist(Histogram::getHistogram(LiteralStringRef("GrvProxy"),
+	                                                    LiteralStringRef("grvConfirmEpochLive"),
```

Comment:
```suggestion
	                                                    LiteralStringRef("GrvConfirmEpochLive"),
```

Reply:
Did something happen to this? I can't tell from the GitHub history whether this and my other suggestion were applied or not.

Reply:
I force pushed my local commit, which overwrote the suggestion. I re-applied the change. Please take a look.